2024-03-09 20:56:43,908 INFO    MainThread:15271 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-09 20:56:43,908 INFO    MainThread:15271 [wandb_setup.py:_flush():76] Configure stats pid to 15271
2024-03-09 20:56:43,908 INFO    MainThread:15271 [wandb_setup.py:_flush():76] Loading settings from /Users/sunayana/.config/wandb/settings
2024-03-09 20:56:43,909 INFO    MainThread:15271 [wandb_setup.py:_flush():76] Loading settings from /Users/sunayana/Documents/GitHub/CV/assignment2/wandb/settings
2024-03-09 20:56:43,909 INFO    MainThread:15271 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-09 20:56:43,909 INFO    MainThread:15271 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-09 20:56:43,909 INFO    MainThread:15271 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-09 20:56:43,909 INFO    MainThread:15271 [wandb_init.py:_log_setup():526] Logging user logs to /Users/sunayana/Documents/GitHub/CV/assignment2/wandb/run-20240309_205643-hfvgzg4o/logs/debug.log
2024-03-09 20:56:43,909 INFO    MainThread:15271 [wandb_init.py:_log_setup():527] Logging internal logs to /Users/sunayana/Documents/GitHub/CV/assignment2/wandb/run-20240309_205643-hfvgzg4o/logs/debug-internal.log
2024-03-09 20:56:43,909 INFO    MainThread:15271 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x2964ddaf0>
2024-03-09 20:56:43,909 INFO    MainThread:15271 [wandb_init.py:init():566] calling init triggers
2024-03-09 20:56:43,909 INFO    MainThread:15271 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-03-09 20:56:43,909 INFO    MainThread:15271 [wandb_init.py:init():616] starting backend
2024-03-09 20:56:43,909 INFO    MainThread:15271 [wandb_init.py:init():620] setting up manager
2024-03-09 20:56:43,916 INFO    MainThread:15271 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-03-09 20:56:43,917 INFO    MainThread:15271 [wandb_init.py:init():628] backend started and connected
2024-03-09 20:56:43,930 INFO    MainThread:15271 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-03-09 20:56:43,931 INFO    MainThread:15271 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-03-09 20:56:43,931 INFO    MainThread:15271 [wandb_init.py:init():720] updated telemetry
2024-03-09 20:56:43,961 INFO    MainThread:15271 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-03-09 20:56:45,317 INFO    MainThread:15271 [wandb_run.py:_on_init():2262] communicating current version
2024-03-09 20:56:46,795 INFO    MainThread:15271 [wandb_run.py:_on_init():2271] got version response upgrade_message: "wandb version 0.16.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-03-09 20:56:46,795 INFO    MainThread:15271 [wandb_init.py:init():804] starting run threads in backend
2024-03-09 20:56:51,527 INFO    MainThread:15271 [wandb_run.py:_console_start():2241] atexit reg
2024-03-09 20:56:51,527 INFO    MainThread:15271 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-03-09 20:56:51,528 INFO    MainThread:15271 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-03-09 20:56:51,528 INFO    MainThread:15271 [wandb_run.py:_redirect():2186] Redirects installed.
2024-03-09 20:56:51,528 INFO    MainThread:15271 [wandb_init.py:init():847] run started, returning control to user process
2024-03-09 20:56:51,571 INFO    MainThread:15271 [wandb_watch.py:watch():51] Watching
2024-03-09 21:00:24,605 INFO    MainThread:15271 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-09 21:00:24,606 INFO    MainThread:15271 [wandb_init.py:_pause_backend():437] pausing backend
2024-03-09 21:10:38,723 INFO    MainThread:15271 [wandb_init.py:_resume_backend():442] resuming backend
2024-03-09 21:10:38,736 INFO    MainThread:15271 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-09 21:10:38,736 INFO    MainThread:15271 [wandb_setup.py:_flush():76] Configure stats pid to 15271
2024-03-09 21:10:38,736 INFO    MainThread:15271 [wandb_setup.py:_flush():76] Loading settings from /Users/sunayana/.config/wandb/settings
2024-03-09 21:10:38,736 INFO    MainThread:15271 [wandb_setup.py:_flush():76] Loading settings from /Users/sunayana/Documents/GitHub/CV/assignment2/wandb/settings
2024-03-09 21:10:38,736 INFO    MainThread:15271 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-09 21:10:38,736 INFO    MainThread:15271 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-09 21:10:38,736 INFO    MainThread:15271 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-09 21:10:38,736 INFO    MainThread:15271 [wandb_init.py:_log_setup():526] Logging user logs to /Users/sunayana/Documents/GitHub/CV/assignment2/wandb/run-20240309_211038-wrult4u4/logs/debug.log
2024-03-09 21:10:38,736 INFO    MainThread:15271 [wandb_init.py:_log_setup():527] Logging internal logs to /Users/sunayana/Documents/GitHub/CV/assignment2/wandb/run-20240309_211038-wrult4u4/logs/debug-internal.log
2024-03-09 21:10:38,736 INFO    MainThread:15271 [wandb_init.py:init():566] calling init triggers
2024-03-09 21:10:38,736 INFO    MainThread:15271 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'batch_size': 32, 'learning_rate': 0.001, 'optimizer_choice': 'Adam'}
2024-03-09 21:10:38,736 INFO    MainThread:15271 [wandb_init.py:init():591] re-initializing run, found existing run on stack: hfvgzg4o
2024-03-09 21:10:38,737 INFO    MainThread:15271 [wandb_run.py:_finish():1970] finishing run sunayana981/lenet_mnist_project/hfvgzg4o
2024-03-09 21:10:38,737 INFO    MainThread:15271 [jupyter.py:save_history():445] not saving jupyter history
2024-03-09 21:10:38,737 INFO    MainThread:15271 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-09 21:10:38,737 INFO    MainThread:15271 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-03-09 21:10:38,738 INFO    MainThread:15271 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-03-09 21:10:38,738 INFO    MainThread:15271 [wandb_run.py:_restore():2193] restore
2024-03-09 21:10:38,738 INFO    MainThread:15271 [wandb_run.py:_restore():2199] restore done
2024-03-09 21:10:48,655 INFO    MainThread:15271 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-03-09 21:10:48,657 INFO    MainThread:15271 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-03-09 21:10:48,663 INFO    MainThread:15271 [wandb_run.py:_footer_sync_info():3825] logging synced files
